languages/elixir/reference/README.md

```diff
@@ -0,0 +1,145 @@
+# Elixir reference
+
+## Concepts
+
+Below are the concepts that have been identified in Elixir for use in Exercism.
+
+### Functional
+
+- [Anonymous functions](../../../reference/concepts/anonymous_functions.md)
+- [Higher-order functions](../../../reference/concepts/higher_order_functions.md)
+- [Immutability](../../../reference/concepts/immutability.md)
+- [Pattern matching](../../../reference/concepts/pattern_matching.md)
+- [Pipelines](../../../reference/concepts/pipelines.md)
+- [Recursion](../../../reference/concepts/recursion.md)
+- Tail Call Optimization
+
+### Platform-specific
+
+- BEAM VM
+- Processes
+- Agent Concurrency Model
+- Erlang Interoperation
```

Comment:
Very happy to see this in here - it's something I think we don't do enough of in the current track!